src/components/Admin/Dashboard/Products.vue

```diff
@@ -229,27 +141,28 @@
       openProduct (pindex) {
         this.productIndex = pindex
         this.currentProduct = this.addedProductsData.results[pindex]
-        this.currentProduct.price_cents = this.currentProduct.price_cents / 100
+        this.newPrice = this.currentProduct.price_cents / 100
         setTimeout(this.$refs.productEdit.open(), 300)
       },
+      closeProductModal () {
```

Comment:
This should update the product info in the actual addedProductsData array.. that is the array I will be sending to you to add the products the user specified